package.json

```diff
@@ -10,7 +10,7 @@
     "url": "shelf.io"
   },
   "engines": {
-    "node": ">=8.10"
+    "node": ">=10"
```

Comment:
Since this PR supports both node 8 & node 10, maybe leave `engines` setting as it was before?